diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/AppDelegate.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/AppDelegate.h
index 17678ea..f2ab59a 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/AppDelegate.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/AppDelegate.h
@@ -1,5 +1,5 @@
 #import <Foundation/Foundation.h>
-#import "CDVViewController.h"
+#import <CapacitorCordova/CDVViewController.h>
 
 @interface AppDelegate : NSObject
 
diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDV.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDV.h
index 8f8fc04..72e350f 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDV.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDV.h
@@ -17,12 +17,12 @@
  under the License.
  */
 
-#import "CDVAvailability.h"
-#import "CDVPlugin.h"
-#import "CDVPluginResult.h"
-#import "CDVCommandDelegate.h"
-#import "CDVInvokedUrlCommand.h"
-#import "CDVViewController.h"
-#import "CDVURLProtocol.h"
-#import "CDVScreenOrientationDelegate.h"
-#import "CDVWebViewProcessPoolFactory.h"
+#import <CapacitorCordova/CDVAvailability.h>
+#import <CapacitorCordova/CDVPlugin.h>
+#import <CapacitorCordova/CDVPluginResult.h>
+#import <CapacitorCordova/CDVCommandDelegate.h>
+#import <CapacitorCordova/CDVInvokedUrlCommand.h>
+#import <CapacitorCordova/CDVViewController.h>
+#import <CapacitorCordova/CDVURLProtocol.h>
+#import <CapacitorCordova/CDVScreenOrientationDelegate.h>
+#import <CapacitorCordova/CDVWebViewProcessPoolFactory.h>
diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegate.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegate.h
index 45869bd..d56df09 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegate.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegate.h
@@ -17,7 +17,7 @@
  under the License.
  */
 
-#import "CDVInvokedUrlCommand.h"
+#import <CapacitorCordova/CDVInvokedUrlCommand.h>
 
 @class CDVPlugin;
 @class CDVPluginResult;
diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegateImpl.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegateImpl.h
index f42654e..3348788 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegateImpl.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVCommandDelegateImpl.h
@@ -18,9 +18,9 @@
  */
 
 #import <UIKit/UIKit.h>
-#import "CDVCommandDelegate.h"
+#import <CapacitorCordova/CDVCommandDelegate.h>
 #import <WebKit/WebKit.h>
-#import "CDVPluginManager.h"
+#import <CapacitorCordova/CDVPluginManager.h>
 
 @class CDVViewController;
 @class CDVCommandQueue;
diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin+Resources.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin+Resources.h
index cc43b16..e367956 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin+Resources.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin+Resources.h
@@ -18,7 +18,7 @@
  */
 
 #import <UIKit/UIKit.h>
-#import "CDVPlugin.h"
+#import <CapacitorCordova/CDVPlugin.h>
 
 @interface CDVPlugin (CDVPluginResources)
 
diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin.h
index 29ba4cd..3b314cd 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPlugin.h
@@ -19,9 +19,9 @@
 
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
-#import "CDVPluginResult.h"
-#import "CDVCommandDelegate.h"
-#import "CDVAvailability.h"
+#import <CapacitorCordova/CDVPluginResult.h>
+#import <CapacitorCordova/CDVCommandDelegate.h>
+#import <CapacitorCordova/CDVAvailability.h>
 #import <WebKit/WebKit.h>
 
 @interface UIView (org_apache_cordova_UIView_Extension)
diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPluginManager.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPluginManager.h
index b67fbba..df6da6d 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPluginManager.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVPluginManager.h
@@ -6,9 +6,9 @@
 //
 
 #import <Foundation/Foundation.h>
-#import "CDVPlugin.h"
-#import "CDVConfigParser.h"
-#import "CDVCommandDelegate.h"
+#import <CapacitorCordova/CDVPlugin.h>
+#import <CapacitorCordova/CDVConfigParser.h>
+#import <CapacitorCordova/CDVCommandDelegate.h>
 
 @interface CDVPluginManager : NSObject
 
diff --git a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVURLProtocol.h b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVURLProtocol.h
index 0561e04..51ee87d 100644
--- a/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVURLProtocol.h
+++ b/node_modules/@capacitor/ios/CapacitorCordova/CapacitorCordova/Classes/Public/CDVURLProtocol.h
@@ -18,7 +18,7 @@
  */
 
 #import <Foundation/Foundation.h>
-#import "CDVAvailability.h"
+#import <CapacitorCordova/CDVAvailability.h>
 
 @class CDVViewController;
 
